src/test/java/com/telefonica/iot/cygnus/backends/ckan/CKANBackendImplTest.java

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
The same answer than "HDFSBackendImpl", this string must be configured by the users in a Cygnus parameter and it would be very disturbing to change it now. Let's live with it!